rental.rb

```diff
@@ -0,0 +1,13 @@
+# rental.rb
+class Rental
+  attr_acessor :date
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+
+    book.add_rental(self)
+    person.add_rental(self)
```

Comment:
- [ Required ] I think it would be nice if you add the rentals for a `book` or `person` object through their `rentals` properties. Since the `rentals` properties in classes `Book` and `Person` are arrays you can push the new `rental` object into the array. Please see the example code below for a suggestion on how to achieve this.

```ruby
class Rental
  attr_acessor :date

  def initialize(date, person, book)
    @date = date
    @person = person
    person.rentals << self

    @book = book
    book.rentals << self
  end
end
```